src/Core/Services/Implementations/SendService.cs

```diff
@@ -43,6 +51,25 @@ public class SendService : ISendService
 
         public async Task SaveSendAsync(Send send)
         {
+            // Make sure user can save Sends
+            if (send.UserId.HasValue)
+            {
+                var policies = await _policyRepository.GetManyByUserIdAsync(send.UserId.Value);
+                if (policies != null)
+                {
+                    foreach (var policy in policies.Where(p => p.Enabled && p.Type == PolicyType.DisableSend))
+                    {
+                        var org = await _organizationUserRepository.GetDetailsByUserAsync(send.UserId.Value, policy.OrganizationId,
+                            OrganizationUserStatusType.Confirmed);
+                        if (org != null && org.Enabled && org.UsePolicies
+                           && org.Type != OrganizationUserType.Admin && org.Type != OrganizationUserType.Owner)
```

Comment:
Need to check Permissions here too, or alternatively use `CurrentContext.ManagePolicies` and remove the `OrganizationUserType` checks entirely. 

```suggestion
                        if (org != null && org.Enabled && org.UsePolicies
                           && !(org.Type == OrganizationUserType.Admin || org.Type == OrganizationUserType.Owner || org.Permissions.ManagePolicies))
```

Reply:
ah! I'd forgotten about your context changes. That's much better, thank you.